TB - 79 Update documentation for kafka and authentication changes (#88)
jembi/openhim-docs: docs/user-guide/adding-users.md

2. List settings: Upon clicking on a transaction in the transactions page, you can choose whether to view the transaction on the same page (default), or to open it in a new window altogether.

If you find a field that is not described here, please let us know by [filing an issue on github](https://github.com/jembi/openhim-core-js/issues/new) with the 'documentation' label.

## Keycloak Single Sign-On (SSO)

Openhim console and core are supporting SSO authentication (Keycloak identity manager).
Enabling SSO in the openhim console and core will allow:

1. Authentication: A user can be logged in using keycloak credentials.

> 2. Profile setup: The user authenticated using Keycloak will have his profile filled automatically with info provided by Keycloak. 

Reviewer: ```suggestion
2. Profile setup: The user authenticated using Keycloak will have a profile created automatically with info provided by Keycloak. 
```
